apps/web/src/lib/components/settings/settings-content.svelte

```diff
@@ -267,10 +273,122 @@
         database.storageSourcesChanged$.next([]);
       });
   }
+
+  let bookName: string = 'Loading...';
+
+  $: presetOptions = bookId ? PresetStorage.getPresetList() : ['Global'];
+  $: presetName = $bookPresets$[bookId.toString()] ?? 'Global';
+  $: presetStorage.setPreset(presetName);
+
+  $: presetDeleteStyling =
```

Comment:
i would prefer class binding on the element vs reactive statement for styles/classes